docs/source/tutorials/colab_requirements.txt

```diff
@@ -0,0 +1,213 @@
+absl-py==0.10.0
+aiohttp==3.7.3
+aiohttp-cors==0.7.0
+aioredis==1.3.1
+alabaster==0.7.12
+appdirs==1.4.4
+APScheduler==3.6.3
+argon2-cffi==20.1.0
+ase==3.19.1
+async-generator==1.10
+async-timeout==3.0.1
+attrs==20.3.0
+Babel==2.9.0
+backcall==0.2.0
+beautifulsoup4==4.9.3
+black==20.8b1
+bleach==3.3.0
+blessings==1.7
+brotlipy==0.7.0
+cached-property==1.5.2
+cachetools==4.2.1
+certifi==2020.6.20
+cffi==1.14.5
+cfgv==3.2.0
+cftime==1.4.1
+chardet==3.0.4
+click==7.1.2
+cloudpickle==1.6.0
+colorama==0.4.4
+colorful==0.5.4
+configparser==5.0.1
+contextvars==2.4
+cryptography==3.4.4
+cycler==0.10.0
+dataclasses==0.7
```

Comment:
In the latest version of google colab, python 3.7.x may be used, so this cause error
> ERROR: Could not find a version that satisfies the requirement dataclasses==0.7

Reply:
Thanks for pointing this out, I actually removed the specific version from the requirements.txt I was using with Colab but did not push that change to GitHub yet so I will do so now